src/sentry/static/sentry/app/views/organizationEventsV2/data.jsx

```diff
@@ -123,6 +118,37 @@ export const SPECIAL_FIELDS = {
       </Container>
     ),
   },
+  error: {
+    title: 'error',
+    groupby: ['issue.id', 'project.id'],
+    aggregations: [['anyHeavy', 'title', 'issue_group']],
```

Comment:
Using https://clickhouse.yandex/docs/en/query_language/agg_functions/reference/#anyheavy-x for now but we probably need to revisit which title we want to show when events are grouped

Reply:
Does the clientside application need this low-level of control over how queries in snuba are created? This feels riskier both from a risk and coupling perspective than having a safe-list of pre-canned queries supported by the API which then translates those canned names into snuba queries.

Reply:
@markstory I had a go at adding some special field support on the API https://github.com/getsentry/sentry/pull/13453/commits/31d2f3ff0a0cbf33e83c42525c34a1a86be3ab8a. Would you consider this syntax to be an improvement? I'm on the fence.

Reply:
The other option would be to simply hardcode the 3 queries on the backend for "all", "errors" and "csp". Since we talked about future versions of this supporting interactive pivot-table like behaviour it might be a bit restrictive though.